content/ios/user.md

```diff
@@ -106,16 +96,31 @@ Sometimes users would want to change their passwords to secure their accounts. T
                  password:@"newPassword"
         completionHandler:^(SKYUserRecordID *user, NSError *error) {
     if (error) {
-        NSLog(@"Can't change password: %@", error);
+        NSLog(@"can't change password: %@", error);
         // Can be old password not matched?
         return;
     }
 
-    NSLog(@"Password change successfully");
+    NSLog(@"password change successfully");
 }]
 ```
 
-## Disocver users by emails
+# Current User
+## Getting current user id
+
+You can get the `currentUserRecordID` by:
+
+```obj-c
+if (container.currentUserRecordID) {
+    // user is logged in, proceed
+} else {
+    // ask user to sign up / login
+}
+```
+
+## Looking up users by email
+
+The following code shows how you can query other users with their emails:
```

Comment:
- "query other users by their emails", but I believe you can directly follows how the JS SDK guide write the sentence. I believe you can refer to the JS counterpart when you are writing sentences that will be common between different SDKs. @TakLee96 please notice this as well.
- Better mention if there is any access restriction, i.e. any user can search for any other emails?